core/client/src/client.rs

```diff
@@ -911,7 +916,7 @@ impl<B, E, Block, RA> CallApiAt<Block> for Client<B, E, Block, RA> where
 	B: backend::Backend<Block, Blake2Hasher>,
 	E: CallExecutor<Block, Blake2Hasher> + Clone + Send + Sync,
 	Block: BlockT<Hash=H256>,
-	RA: CoreAPI<Block>
+	RA: Send + Sync,
```

Comment:
```suggestion
	RA: CoreAPI<Block>,
```
This should not be changed!

Reply:
why? everything compiles without?

Reply:
Yeah it compiles without it, but it enforces that the `CoreAPI` is implemented. The `CoreAPI` is required to be implemented by each runtime. It internally also uses a function from the `CoreAPI`, the `initialise_block` function. This function is currently not called by the corresponding trait function, but putting the `CoreAPI` as an requirement, makes sure the function is present.

Reply:
ok, i will re-introduce and add a comment to that effect.